plugins/dataframe-gradle-plugin/src/main/kotlin/org/jetbrains/dataframe/gradle/ConvenienceSchemaGeneratorPlugin.kt

```diff
@@ -67,19 +67,23 @@ class ConvenienceSchemaGeneratorPlugin : Plugin<Project> {
                     isMultiplatform -> listOf("kspJvm", "kspJvmTest")
                     else -> listOf("ksp", "kspTest")
                 }
+
+                val cfgsToAdd = configs.toMutableSet()
+
                 configs.forEach { cfg ->
-                    try {
-                        target.configurations.getByName(cfg).dependencies.add(
-                            target.dependencies.create(
-                                "org.jetbrains.kotlinx.dataframe:symbol-processor-all:$preprocessorVersion",
-                            ),
-                        )
-                    } catch (e: UnknownConfigurationException) {
-                        target.logger.warn(
-                            "Configuration '$cfg' not found. Please make sure the KSP plugin is applied.",
+                    target.configurations.named { it == cfg }.configureEach {
```

Comment:
It may be clearer to use `.findByName(cfg)` (which is the nullable version of `.getByName()`) I was a bit confused by `configureEach` since there's no possibility there are ever multiple configurations with the same name, are there?

Reply:
Alright,thanks for the clarification :)